src/task_scl_eff_pot_hab.py

```diff
@@ -48,9 +47,14 @@ class SCLPolygons(SCLTask):
             "ee_path": "projects/SCL/v1/Panthera_tigris/source/Inputs_2006/eco_cntry",
```

Comment:
this should point to `USDOS/LSIB/2013` and then we rasterize as part of task
For this and for ecoregions, we should both try and think of a way to have tasks share common definitions